### PR TITLE
Final Multiselect widget migration

### PR DESCRIPTION
**Pull request by biyeun**
- replaces old multiselect widget with the new one from https://github.com/dimagi/commcare-hq/pull/11760
- This needs https://github.com/dimagi/commcare-hq/pull/11760 to be merged before this, but I figured I'd put it up for now
- review commit-by-commit after d2a222d

@orangejenny @benrudolph cc @czue @dimagi/product 

screen shot 2016-05-26 at 2 32 01 pm
screen shot 2016-05-26 at 2 23 06 pm
screen shot 2016-05-26 at 2 18 37 pm
screen shot 2016-05-26 at 1 54 44 pm

### CODE REVIEW

**Comment by biyeun** [comment]
since the old multiselects are actually broken on B3 pages (drag + drop), and this widget has been reliable in the hallway testing (and local QA), this should be g2m after ✅

File: corehq/apps/accounting/views.py [hunk 1]
```diff
@@ -230,6 +232,10 @@ class NewSubscriptionView(AccountingSectionView, AsyncHandlerMixin):
         Select2BillingInfoHandler,
     ]
 
+    @use_jquery_ui  # for datepicker
```
Comment (orangejenny) [inline corehq/apps/accounting/views.py]: appreciate the comment

**Comment by orangejenny** [comment]
Good to merge pending tests.

Can you also delete the multiselect files (`ui.multiselect.css`, `jquery-ui-1.9.2.multiselect-deps.custom.min.js`, `ui.multiselect.js`)?

This can also probably be deleted: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/static/style/less/bootstrap3/forms.less#L164-L166

**Comment by orangejenny** [comment]
https://github.com/dimagi/commcare-hq/pull/11760 got merged - there's nothing else blocking merge of this, right?

**Comment by biyeun** [comment]
@orangejenny nope nothing else blocking the merge, i'll go ahead and make those updates

**Comment by biyeun** [comment]
@orangejenny i'll clean up the `.less` file in the final migration...

**Comment by orangejenny** [comment]
cool, good to merge pending tests

**Comment by biyeun** [comment]
argh...build timeout again.

**Pull request merged**